appveyor.yml

```diff
@@ -13,9 +13,7 @@ environment:
 
       NUMPY_VERSION: "stable"
       CONDA_DEPENDENCIES: "pyqt numpy astropy glue-core>=0.13 specviz=0.5"
-
-      # N.B regions package: need the work in master as of 9-11-2018 which has not yet been released
-      PIP_DEPENDENCIES: "pytest-qt git+https://github.com/astropy/regions.git"
+      PIP_DEPENDENCIES: "pytest-qt"
```

Comment:
Why not include ``regions>=0.3`` here?

Reply:
Gah, an oversight. Although, to be fair, my original intent was to test the installation of dependencies during `setup.py develop/install`, which very few of our packages actually do...